### ISSUE TITLE
Remove scipy from code?

### ISSUE THREAD

**Issue by hagenw** [audeering/audtorch#66]
With not including `librosa` in the dependencies we had the problem of #50.
In the end, we added it to `setup.cfg`.

We still have a hidden dependency on `scipy`, which is not listed in `setup.cfg`, and I'm reluctant to add it there. It is used as a non-default option in `transforms.Resample`. My question is: should we just stay with it, or maybe remove the `scipy`  resample option altogether as it can crash as well?

**Comment by ATriantafyllopoulos** [comment]
If I remember correctly this was the fastest options for small files, right?

**Comment by hagenw** [comment]
Yes, it was. But at the moment the recommended workflow is to do the resampling first anyway.
